src/seml/experiment/config.py

```diff
@@ -561,8 +631,8 @@ def resolve_configs(
     exp = exps[0]
     if not isinstance(exp, Experiment):
         logging.warning(
-            'The use of sacred.Experiment is deprecated. Please use seml.experiment.Experiment instead.\n'
-            'seml.experiment.Experiment already includes typical MongoDB observer and logging setups.\n'
+            'The use of sacred.Experiment is deprecated. Please use seml.Experiment instead.\n'
```

Comment:
Inconsistent, `seml.Experiment` is neither usable as subclass nor as type annotation.